cmd/bosun/sched/host.go

```diff
@@ -440,6 +440,24 @@ func (s *Schedule) Host(filter string) (map[string]*HostData, error) {
 					if iface != nil {
 						iface.Description = val
 					}
+				case "dataStores":
+					dataStores := []string{}
+					err = json.Unmarshal([]byte(val), &dataStores)
+					if err != nil {
+						slog.Errorf("error unmarshalling datastores for host %s while generating host api: %s", host.Name, err)
```

Comment:
@captncraig The code makes it sounds like it checks for this first, see https://golang.org/src/encoding/json/decode.go?s=2621:2669#L64